### ISSUE TITLE
Beginning of waterfall is missing for iphone tests

### ISSUE THREAD

**Issue by connorjclark** [WPO-Foundation/webpagetest#1336]
https://www.webpagetest.org/result/200226_8V_2cc5ddef4b9e419830a3945f02d25d87/1/details/#waterfall_view_step1

I tried for iphone 7 and 8. Notice that the first request is not the initial document url (it never shows up). also, no css is loaded. The beginning of the waterfall seems to have been cut off.

This is not a repeat view so it isn't b/c of cache.

This test on an android device for the same URL shows everything correctly: https://www.webpagetest.org/result/200226_MW_a2fc9bd567a306a2d2bafd4d4291c0a6/1/details/#waterfall_view_step1

**Comment by pmeenan** [comment]
Yep, sorry - iOS 12.3 changed the devtools protocol in a way that makes it impossible to get the initial request (and there may be other things that need to be looked into).  Here is the issue tracking it: https://github.com/WPO-Foundation/wptagent/issues/265
